src/components/autocomplete/autocomplete.spec.js

```diff
@@ -98,6 +98,31 @@ describe('<md-autocomplete>', function() {
       element.remove();
     }));
 
+    it('should allow receiving focus on the autocomplete', function() {
+      var scope = createScope(null, {inputId: 'custom-input-id'});
+      var template = '\
```

Comment:
Yeah- typically good, but I've been hoping to eliminate the `\` stuff over time.